jest.config.mjs

```diff
@@ -11,6 +11,8 @@ const config = {
   // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
 
   testEnvironment: "jest-environment-jsdom",
+  setupFilesAfterEnv: ["<rootDir>/__tests__/setupTests.js"],
+  testPathIgnorePatterns: ["<rootDir>/build/", "<rootDir>/node_modules/", "<rootDir>/__tests__/setupTests.js"]
```

Comment:
Interested to know how you arrived at these values. the `setupTests.js` seems fairly obvious, but the other ones are a bit interesting. Are those defaults that now need to be specified since we are including this config, or are those ones that are usually added and just where not added originally because I set it up and still learning 😅